README.md

```diff
@@ -36,10 +36,10 @@ use ButterCMS\ButterCMS;
 $butterCms = new ButterCMS('<auth_token>');
 
 // Feeds - returns a SimpleXMLElement object
-$feed = $butterCms->getFeed('rss');
+$feed = $butterCms->fetchFeed('rss');
 
 // Posts
-$result = $butterCms->getPosts(['page' => 1]);
+$result = $butterCms->fetchPosts(['page' => 1]);
```

Comment:
Hey, I will have time to actually commit / test this later, but the code below should show how to do that. Not sure if you want `<br/>` or `PHP_EOL` after each post.

```
$response = $butterCms->fetchPosts(['page' => 1]);
foreach ($response->getPosts() as $post) {
    echo $post->getTitle();
}
```